contrib/lang/clojure/packages.el

```diff
@@ -77,7 +77,7 @@ which require an initialization must be listed explicitly in the list.")
         "mgv" 'cider-jump-to-var
         "mgr" 'cider-jump-to-resource
         "mge" 'cider-jump-to-compilation-error
-        "mgs" 'cider-jump
```

Comment:
Ah, I was probably thinking of the `mgg` convention specified here: https://github.com/syl20bnr/spacemacs/blob/master/doc/CONVENTIONS.md#code-navigation